src/test/java/com/snowflake/kafka/connector/records/HeaderTest.java

```diff
@@ -194,6 +195,22 @@ public void testTypes() throws IOException
 
   }
 
+  @Test
+  public void testEmptyMap()throws IOException {
```

Comment:
Can we include a test with a map with values?

Reply:
The original test in this file had a case for map schema